taccsite_cms/static/site_cms/css/src/_imports/trumps/s-document.css

```diff
@@ -41,15 +41,6 @@ Styleguide Trumps.Scopes.Document
   blockquote { width: 600px; }
 }
 
-.s-document figure {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
-}
-.s-document figcaption {
-  font-weight: bold;
-}
-
```

Comment:
I delete redundant code.

<details>

1. This code is already in `s-guide-doc.css`.
1. [Guide pages load `s-document` and `s-guide-doc` styles](https://github.com/TACC/Core-CMS/blob/a032b48/taccsite_cms/static/site_cms/css/src/template.guide.css#L8-L9).
1. `s-document` class is only added on by https://github.com/TACC/Core-CMS/blob/a032b48/taccsite_cms/templates/guide.html, https://github.com/TACC/Core-CMS/blob/d6927b8/taccsite_cms/templates/home_portal.html, and https://github.com/TACC/Core-CMS/blob/92e577cdaf86bc7473b72d1ef7409e0d71917571/taccsite_cms/templates/style_guide.html.
    - `guide`, relevant—it's for Guide pages.
    - `home_portal`, irrelevant—it does not use `<figure>`'s ([proof](https://cep.tacc.utexas.edu/)).
    - `style_guide`, irrelevant—unofficially deprecated and unused test template.
1. If only `guide` is relevant, then only `s-guide-doc` needs that code.

Screenshots
- [duplicate CSS code](https://user-images.githubusercontent.com/62723358/157769043-1d69533a-2f65-49c4-a959-6d754ab7344f.png)
- [`s-document` search (sans css files)](https://user-images.githubusercontent.com/62723358/157769048-66b581c0-ce7d-45fc-89ec-04a3d83a1b61.png)
- [`s-guide-doc` search (sans css files)](https://user-images.githubusercontent.com/62723358/157769049-cad3521c-c409-4b99-89ef-06c8bb4b7dcf.png)

_P.S. Yes, the docs page CSS organization was over-engineered. I didn't realize the guide pages were one-off's that would eventually be replaced by one new TACC docs. So, I had planned for different kinds of documentation pages on CMS. When guide pages go, so can these stylesheets be removed or simplified._

</details>

Reply:
If it still looks like I might break stuff, just lemme know you're worried. I'll move this bit to another PR, no problem.